### ISSUE TITLE
Effect Not Working(Only Works On Double Tap)

### ISSUE THREAD

**Issue by clifzero** [mrxten/XamEffects#15]
Hi 

As you can see above my issue is the effect only works when I double tap a layout(stacklayout). I tried creating a fresh project with one page thats exactly the same as the one that the plugin is not working on, except, the plugin works on that project. The only thing I can think of is, maybe your plugin is clashing with another plugin that I have installed in my project, for that reason I just wanted to show you all the packages I have installed and maybe you can tell me if you notice some familiar, problematic  
plugins

```
<PackageReference Include="Newtonsoft.Json">
      <Version>11.0.2</Version>
    </PackageReference>
    <PackageReference Include="Plugin.Share">
      <Version>7.1.1</Version>
    </PackageReference>
    <PackageReference Include="Prism.Core">
      <Version>6.3.0</Version>
    </PackageReference>
    <PackageReference Include="Prism.Forms">
      <Version>6.3.0</Version>
    </PackageReference>
    <PackageReference Include="Prism.Unity.Forms">
      <Version>6.3.0</Version>
    </PackageReference>
    <PackageReference Include="Rg.Plugins.Popup">
      <Version>1.1.4.158-pre</Version>
    </PackageReference>
    <PackageReference Include="Xam.Plugins.Settings">
      <Version>3.1.1</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.Facebook.iOS">
      <Version>4.24.0</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.FFImageLoading">
      <Version>2.3.6</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.FFImageLoading.Forms">
      <Version>2.3.6</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.FFImageLoading.Transformations">
      <Version>2.3.6</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.Firebase.iOS.Analytics">
      <Version>4.0.5</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.Firebase.iOS.CloudMessaging">
      <Version>2.0.8</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.Firebase.iOS.Core">
      <Version>4.0.13</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.Firebase.iOS.InstanceID">
      <Version>2.0.8</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.Forms" Version="2.5.0.280555" />
    <PackageReference Include="ZXing.Net.Mobile">
      <Version>2.4.1</Version>
    </PackageReference>
    <PackageReference Include="ZXing.Net.Mobile.Forms">
      <Version>2.4.1</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.Google.iOS.SignIn">
      <Version>4.1.1.1</Version>
    </PackageReference>
    <PackageReference Include="XamEffects">
      <Version>1.5.0</Version>
    </PackageReference>
  </ItemGroup>
```

thanks, much appreciated

**Comment by mrxten** [comment]
Could you send me your test project?

**Comment by clifzero** [comment]
Sure no problem I'll upload it and send you a link

**Comment by clifzero** [comment]
[Here](https://drive.google.com/open?id=1mC2cMMZ1MxBGQEr8h2RmpPWB-61FnoVB) you go :)

**Comment by mrxten** [comment]
Ok will check tomorrow.

**Comment by mrxten** [comment]
@Cliffy786 Hi, I tested your sample in android.
Works with ONE tap...
In which android version do you tested?

**Comment by clifzero** [comment]
There's no problem with the sample, the sample works perfectly, the actual
app that I was developing has the double tap issue.

**Comment by mrxten** [comment]
Without sources I cannot help you

**Comment by clifzero** [comment]
I can't give you my source code, thats why I provided you with the plug-ins
that I'm using in my project, because I thought maybe, by some small
chance, that there might be a plug in I'm using that you might have known
of thats notorious for clashing with your plug in and causing bugs.

**Comment by mrxten** [comment]
Check pls do you use same versions  XF and support libraries in main project and sample

**Comment by clifzero** [comment]
Hi

Sorry for taking so long to reply

Xamarin.Forms Versions:


Sandbox 3.0.0.482510

Application 2.5.0.121934


Xamarin.Android.Support Libraries Versions:


Support Libraries:

Sandbox : 27.0.2

Application: 25.4.0.1

**Comment by mrxten** [comment]
Should be fixed with 1.6.0-pre
